tests/unit/test.program.js

```diff
@@ -175,7 +178,7 @@ describe('program.Program', () => {
     });
     program.command('thing', 'does a thing', () => {});
 
-    return execProgram(program, {logStream})
+    return execProgram(program, {getVersion: version, logStream})
```

Comment:
it looks like this can just be `getVersion: spy()` since the `version` variable isn't used elsewhere